packages/ods/src/components/tooltip/src/components/ods-tooltip/ods-tooltip.tsx

```diff
@@ -11,9 +11,14 @@ import { ODS_TOOLTIP_STRATEGY, type OdsTooltipStrategy } from '../../constants/t
 export class OdsTooltip {
   private arrowElement?: HTMLElement;
   private triggerElement?: HTMLElement | null;
+  private isTooltipHover: boolean = false;
+  private timer?: NodeJS.Timeout;
   private cleanUpCallback: () => void = () => {};
   private boundHide = this.hide.bind(this);
   private boundShow = this.show.bind(this);
+  private boundShowOnTooltip = this.showOnElEnter.bind(this);
+  private boundHideOnTooltip = this.hideOnElLeave.bind(this);
```

Comment:
try to keep the name consistent so that it's more easy to know which method `boundXXX` are related to without having to check here
ex: 
```
private boundShowOnTooltip = this.showOnTooltip.bind(this);
// or
private boundShowOnElEnter = this.showOnElEnter.bind(this);
```